setup-environment-internal

```diff
@@ -264,6 +264,7 @@ DISTRO_DIRNAME=$(echo "${DISTRO}" | sed 's#[.-]#_#g')
 
 if [ -z "$LMP_VERSION_CACHE" ]; then
     LMP_VERSION_CACHE="$(git --git-dir ${MANIFESTS}/.git describe HEAD --tags --abbrev=0)"
+    LMP_VERSION_CACHE="$(echo $LMP_VERSION_CACHE | sed 's/\.[0-9]*$//')"
```

Comment:
The only note: I'd use `'s/\.[0-9]+$//'` - there is no option to have a dot without a digit :)